### PR TITLE
Problem: export state & import as genesis (fix #510)

### LINKED ISSUE #510: Problem: export state & import as genesis
this error occurred
```
Error: error validating genesis file
 $HOME/.ethermint_9000-2/config/genesis.json:
 invalid genesis account 0x......................................: 
code cannot be empty
```

### PR DESCRIPTION
**Pull request by leejw51crypto**
Solution: in GenesisAccount, make empty Code legitimate

fixes: #510

### CODE REVIEW

File: x/evm/types/genesis.go [hunk 1]
```diff
@@ -13,9 +11,8 @@ func (ga GenesisAccount) Validate() error {
 	if err := ethermint.ValidateAddress(ga.Address); err != nil {
 		return err
 	}
-	if len(ethcmn.Hex2Bytes(ga.Code)) == 0 {
-		return errors.New("code cannot be empty")
-	}
+	// Code:    []byte{}, like go-ethereum
+	// empty Code is ok
```
Comment (fedekunze) [inline x/evm/types/genesis.go:15]: please remove the comment

Comment (leejw51crypto) [reply x/evm/types/genesis.go:15]: ok, i'll remove it

**Review by fedekunze — APPROVED** [review]

**Comment by fedekunze** [comment]
@leejw51crypto can you fix the test?

**Comment by leejw51crypto** [comment]
sure, i'll fix the test

**Pull request merged**